expected/poly.out

```diff
@@ -349,6 +349,8 @@ LINE 1: SELECT spoly '{(10d,0d),(10d,1d)}';
                      ^
 SELECT spoly(ARRAY[1.0, 2.0, 3.0, 4.0, 5.0]);
 ERROR:  spherepoly_rad: invalid number of arguments (must be even and >= 6)
+SELECT spoly(ARRAY[1.0, 2.0, 3.0, NULL, 5.0, 6.0]);
+ERROR:  spherepoly_rad: invalid array has null values
```

Comment:
Not sure, that it is a good sentence. May be something like it: The input array is invalid because it contains null values? Anyway, I'm not a native speaker. It is up to you.

Reply:
I think my error message was OK, but your version is clearer, so I have changed it to your version.